modules/core/client/directives/page-title.client.directive.js

```diff
@@ -19,13 +19,17 @@
 
       function listener(event, toState) {
         var applicationCoreTitle = 'MEAN.js',
-          separeteBy = ' - ';
+          separeteBy = ' - ',
```

Comment:
pre-existing typo... should be "separate". This should be fixed in these changes.